openshift/nise-populator.yaml

```diff
@@ -172,6 +172,21 @@ objects:
                       key: insights-org-id
                       name: ${NAME}-secret
                       optional: false
+                - name: GCP_DATASET
```

Comment:
Don't update these, update the ones in the clowder one instead: https://github.com/project-koku/nise-populator/blob/main/deploy/clowdapp.yaml